test/test.keystore.claimtx

```diff
@@ -1 +1 @@
-{"address":"0d36c07e49a80f2c84f09ed89dc9e002a29e2a2f","crypto":{"cipher":"aes-128-ctr","ciphertext":"8b1607f33974bb3a11d1e5bcd36c8b18a329b8a8990800d8c626bb449c8605e5","cipherparams":{"iv":"ec2268b7e4cb77f54525ac688c36c8db"},"kdf":"scrypt","kdfparams":{"dklen":32,"n":262144,"p":1,"r":8,"salt":"a78dd695e7c4172b040c9cf71acb0e642ad162915421c92e2deb212f0cbae713"},"mac":"7105e4b4b968665a8fa37eaaf55fba2703ba2e9139cfc926a5fff3cfc2964290"},"id":"2c258b37-c94e-4b6d-9450-ccdaf76fcfad","version":3}
\ No newline at end of file
+{"address":"f39fd6e51aad88f6f4ce6ab8827279cfffb92266","crypto":{"cipher":"aes-128-ctr","ciphertext":"d005030a7684f3adad2447cbb27f63039eec2224c451eaa445de0d90502b9f3d","cipherparams":{"iv":"dc07a54bc7e388efa89c34d42f2ebdb4"},"kdf":"scrypt","kdfparams":{"dklen":32,"n":262144,"p":1,"r":8,"salt":"cf2ec55ecae11171de575112cfb16963570533a9c46fb774473ceb11519eb24a"},"mac":"3eb180d405a5da6e462b2adc00091c14856c91d574bf27348714506357d6e177"},"id":"035454db-6b6d-477f-8a79-ce24c10b185f","version":3}
```

Comment:
I think we should avoid this address, this one is used in the sequencer and it may lead us to nonce conflicts

Reply:
No, we won't have any nonce conflict because the sequencer user the account in L1 and we are using it to send txs in L2. Both layers have different nonce counter

Reply:
ah, right